packages/gatsby-source-wordpress/src/steps/create-schema-customization/helpers.js

```diff
@@ -2,6 +2,50 @@ import store from "~/store"
 import { typeDefinitionFilters } from "./type-filters"
 import { getPluginOptions } from "~/utils/get-gatsby-api"
 import { cloneDeep, merge } from "lodash"
+import { getGatsbyNodeTypeNames } from "../source-nodes/fetch-nodes/fetch-nodes"
+
+export const buildInterfacesListForType = type => {
+  let shouldAddNodeType = false
+
+  const list = (type?.interfaces || [])
+    .filter(interfaceType => {
+      const interfaceTypeSettings = getTypeSettingsByType(interfaceType)
+
+      return (
+        !interfaceTypeSettings.exclude && fieldOfTypeWasFetched(interfaceType)
+      )
+    })
+    .map(({ name }) => {
+      if (name === `Node`) {
+        shouldAddNodeType = true
+      }
+      return buildTypeName(name)
+    })
+
+  if (shouldAddNodeType) {
+    list.push(`Node`)
+  }
+
+  return list
+}
+
+let ceIntCache = null
+const isWpgqlOneThirteenZeroPlus = () => {
+  if (ceIntCache !== null) {
+    return ceIntCache
+  }
+
+  const { typeMap } = store.getState().remoteSchema
+
+  const connectionInterface = !!typeMap.get(`Connection`)
+  const edgeInterface = !!typeMap.get(`Edge`)
+
+  const isWpgqlOneThirteenZeroPlus = connectionInterface || edgeInterface
+
+  ceIntCache = isWpgqlOneThirteenZeroPlus
+
+  return isWpgqlOneThirteenZeroPlus
+}
```

Comment:
I added this helper so we can only make some changes when on the new version of wpgql